it/typescript/src/app/app.ts

```diff
@@ -3,7 +3,7 @@ import { Logger } from './common/logger';
 import { DeviceCodeCredential } from '@azure/identity';
 import { FetchRequestAdapter } from '@microsoft/kiota-http-fetchlibrary';
 import { AzureIdentityAuthenticationProvider } from '@microsoft/kiota-authentication-azure';
-import { type ApiClient } from './client/apiClient';
```

Comment:
this was broken afaict, not caught because we have not been typechecking the output